src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Dropdown.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import Icon from '../../components/Icon';
+import dropdownStyles from './dropdown.scss';
+
+export default class Dropdown extends React.PureComponent {
+    static defaultProps = {
+        isOpen: false,
+        onItemSelected: () => {}
+    };
+
+    handleClick(item) {
+        const {onClick} = item;
+        const {onSelected} = this.props;
+
+        if (onClick) {
+            onClick();
+        }
+
+        onItemSelected(item);
+    }
+
+    render() {
+        const {isOpen, items} = this.props;
+        const dropdownContainerClasses = classNames({
+            [dropdownStyles.container]: true,
+            [dropdownStyles.isOpen]: isOpen,
+        });
+
+        return (
+            <div className={dropdownContainerClasses}>
+                <ul className={dropdownStyles.items}>
```

Comment:
Maybe check how other dropdown buttons in other libraries are implemented. I've started the toolbar component as a unordered list, but had to change it because of @chirimoya 😉